server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider using a descriptive variable name instead of just 'PORT'. This can improve code readability and maintainability.